dd-library-php-setup.php

```diff
@@ -775,136 +780,311 @@ function build_known_command_names_matrix()
+            'description' => 'Path to the request init hook (set by the installer, do not change it)',
+        ],
+        [
+            'name' => 'datadog.trace.enabled',
+            'default' => 'On',
+            'commented' => true,
+            'description' => 'Enables or disables tracing. On by default',
+        ],
+        [
+            'name' => 'datadog.trace.cli_enabled',
+            'default' => 'Off',
+            'commented' => true,
+            'description' => 'Enable or disable tracing of CLI scripts. Off by default',
+        ],
+        [
+            'name' => 'datadog.trace.auto_flush_enabled',
+            'default' => 'Off',
+            'commented' => true,
+            'description' => 'For long running processes, this setting has to be set to On',
+        ],
+        [
+            'name' => 'datadog.trace.generate_root_span',
+            'default' => 'On',
+            'commented' => true,
+            'description' => 'For long running processes, this setting has to be set to Off',
+        ],
+        [
+            'name' => 'datadog.trace.debug',
+            'default' => 'Off',
+            'commented' => true,
+            'description' => 'Enables or disables debug mode.  When On logs are printed to the error_log',
+        ],
+        [
+            'name' => 'datadog.trace.startup_logs',
+            'default' => 'On',
+            'commented' => true,
+            'description' => 'Enables startup logs, including diagnostic checks',
+        ],
+        [
+            'name' => 'datadog.service',
+            'default' => 'my_service',
+            'commented' => true,
+            'description' => 'Sets a custom service name for the application',
+        ],
+        [
+            'name' => 'datadog.env',
+            'default' => 'my_env',
+            'commented' => true,
+            'description' => 'Sets a custom environment name for the application',
+        ],
+        [
+            'name' => 'datadog.version',
+            'default' => '1.0.0',
+            'commented' => true,
+            'description' => 'Sets a version for the user application, not the datadog php library',
+        ],
+        [
+            'name' => 'datadog.agent_host',
+            'default' => '127.0.0.1',
+            'commented' => true,
+            'description' => 'Configures the agent host. If you need more flexibility use `datadog.trace.agent_url` instead',
+        ],
+        [
+            'name' => 'datadog.trace.agent_port',
+            'default' => '8126',
+            'commented' => true,
+            'description' => 'Configures the agent port. If you need more flexibility use `datadog.trace.agent_url` instead',
+        ],
+        [
+            'name' => 'datadog.dogstatsd_port',
+            'default' => '8125',
+            'commented' => true,
+            'description' => 'Configures the dogstatsd agent port. If you need more flexibility use `datadog.trace.agent_url` instead',
```

Comment:
Does the dogstatsd code actually use `datadog.trace.agent_url`? I don't think so, but it's been a while and could be wrong.

Reply:
Your are right https://github.com/DataDog/dd-trace-php/blob/271cff838c370ce06989e0fa89ab1cf67668375c/ext/php7/dogstatsd_client.c#L30-L31 very good catch. Removed it.